internal/levee.go

```diff
@@ -118,3 +123,16 @@ func getArgumentPropagator(c *config.Config, call *ssa.Call) ssa.Node {
 
 	return nil
 }
+
+func isFieldPropagator(c *ssa.Call, fieldPropagators map[types.Object]bool) bool {
+	for o, _ := range fieldPropagators {
+		cf, ok := c.Call.Value.(*ssa.Function)
+		if !ok {
+			continue
+		}
+		if cf.Object() == o {
+			return true
+		}
```

Comment:
This doesn't need to loop over the map.  Go maps return the zero-value when accessing a key outside the map, so you could skip the loop with

```golang
func isFieldPropagator(c *ssa.Call, fieldPropagators map[types.Object]bool) bool {
  cf, ok := c.Call.Value.(*ssa.Function)
  return ok && fieldPropagators[cf.Object()]
}

If `cf.Object()` isn't in the map, the map returns the `bool` zero-value, which is `false`.
```

Reply:
Very nice! Thanks, reads much better.